docs/proposals/15-manage-bastions-and-ssh-key-pair-rotation.md

```diff
@@ -0,0 +1,175 @@
+## Motivation
```

Comment:
Please use [the GEP template](https://github.com/gardener/gardener/blob/master/docs/proposals/00-template.md) including metadata and TOC, and read through the [proposal process description](https://github.com/gardener/gardener/blob/master/docs/proposals/README.md) carefully to familiarize yourself with the process, especially regarding the "main reviewers" concept.

Please try to find a "main reviewer" for your enhancement.